src/api/app/controllers/concerns/rescue_authorization_handler.rb

```diff
@@ -43,6 +43,8 @@ def authorization_message(exception)
       case exception.reason
       when :anonymous_user
         'Please login to access the resource'
+      when :request_state_change
+        "Request #{exception.record.number} would not be acceptable by you"
```

Comment:
A request state change means someone is potentially declining a request, not only accepting it. This message is confusing. How about this?

```suggestion
        "You cannot accept nor decline the request #{exception.record.number}."
```

Reply:
This only ever happens in accepting staging projects

Reply:
I can rename the `.reason` to be more staging related if you want to